### PR TITLE
chore: arkeo route remove

### LINKED ISSUE #6855: Remove arkeo airdrop page
https://app.shapeshift.com/#/arkeo currently is misleading since the snapshot is now over.  

AC

- remove route and related code

### PR DESCRIPTION
**Pull request by twblack88**
## Description

Removes the arkeo route due to a closed snapshot and, as yet, still unannounced launch date. 

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [x] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

closes #6855 

## Risk
low

## Testing

run local and see arkeo route gone

### Operations

same on dev
## Screenshots (if applicable)

Screenshot 2024-06-24 at 10 07 13 PM

### CODE REVIEW

File: src/assets/translations/ja/main.json [hunk 1]
```diff
@@ -2125,7 +2125,7 @@
       }
     },
     "whoQualifies": {
-      "title": "Arkeo Airdropの参加資格",
```
Comment (NeOMakinG) [inline src/assets/translations/ja/main.json:2128]: We need to remove all these copies

File: src/assets/translations/ru/main.json [hunk 2]
```diff
@@ -2112,56 +2112,56 @@
...
-        "3": "ShapeShift занималась развитием проекта под названием Arkeo. Запуск мейннета уже не за горами!",
-        "4": "Arkeo был создан как децентрализованный бэк-энд для всех данных нод. Приложения, использующие Arkeo, получат преимущества в виде увеличения скорости разработки, устойчивости к цензуре и",
+        "3": "ShapeShift занималась развитием проекта под названием . Запуск мейннета уже не за горами!",
+        "4": " был создан как децентрализованный бэк-энд для всех данных нод. Приложения, использующие , получат преимущества в виде увеличения скорости разработки, устойчивости к цензуре и",
         "5": "коэффициента накамото",
         "6": "при этом снижая зависимость от централизованных источников данных."
       }
     },
     "whoQualifies": {
-      "title": "Кто имеет право на получение Arkeo Airdrop?",
```
Comment (NeOMakinG) [inline src/assets/translations/ru/main.json:2128]: We need to remove all these copies

File: src/components/Layout/Header/SideNavContent.tsx [hunk 3]
```diff
@@ -3,8 +3,6 @@ import type { FlexProps } from '@chakra-ui/react'
 import { Box, Flex, IconButton, Stack, useMediaQuery } from '@chakra-ui/react'
 import { lazy, memo, Suspense, useCallback, useMemo } from 'react'
 import { useTranslate } from 'react-polyglot'
-import { useHistory } from 'react-router'
-import { ArkeoIcon } from 'components/Icons/Arkeo'
```
Comment (NeOMakinG) [inline src/components/Layout/Header/SideNavContent.tsx:7]: I think we can also remove it from the project

**Review by NeOMakinG — CHANGES_REQUESTED** [review]
There are a few things that needs to be removed:

- REACT_APP_FEATURE_ARKEO_AIRDROP environment variables
- Everything linked to this environment var: feature flags store keys (ArkeoAirdrop)
- The arkeo cloudflare environment and every thing linked to it in the github ci files and scripts: https://arkeo.shapeshift.com/#/connect-wallet?returnUrl=/dashboard

File: src/components/Layout/Header/SideNavContent.tsx (hunk 3 above)
Comment (twblack88) [reply src/components/Layout/Header/SideNavContent.tsx:7]: the icon? yeah totally. I left some artifacts but maybe it's better to remove all.

File: src/assets/translations/ja/main.json (hunk 1 above)
Comment (twblack88) [reply src/assets/translations/ja/main.json:2128]: done

File: src/assets/translations/ru/main.json (hunk 2 above)
Comment (twblack88) [reply src/assets/translations/ru/main.json:2128]: thanks for the catch!

**Comment by twblack88** [comment]
> * REACT_APP_FEATURE_ARKEO_AIRDROP environment variables
yep done

> * Everything linked to this environment var: feature flags store keys (ArkeoAirdrop)
i just set the flag to false. but good point @NeOMakinG we still have yern flags and a bunch of other things we could just remove. 
 
> * The arkeo cloudflare environment and every thing linked to it in the github ci files and scripts: https://arkeo.shapeshift.com/#/connect-wallet?returnUrl=/dashboard
This I could use a sanity check from @kaladinlight . Are these essential for walletconnecting out at some point or important for users to connect to the chain (should it ever launch?)

**Review by NeOMakinG — APPROVED** [review]

**Pull request merged**